packages/server/utils/sendToSentry.ts

```diff
@@ -11,10 +11,9 @@ export interface SentryOptions {
 }
 
 // Even though this is a promise we'll never need to await it, so we'll never need to worry about catching an error
-const sendToSentry = async (error: Error, options: SentryOptions = {}) => {
-  // if (!PROD) {
+const sendToSentry = async (err: Error | unknown, options: SentryOptions = {}) => {
```

Comment:
let's remove `unknown` from the input union
Some objects don't serialize & if that happens we'll be left with an empty error message

getting into the weeds, `JSON.stringify` simply calls an objects `toJSON` method (or the prototypical parent)
That means I can break it real easy (and many libs do, for good reasons)

e.g.
```ts
JSON.stringify({error: 'very important', toJSON: () => 'eat me'})
```

Reply:
Ahh that's good to know! I was trying to retain the error and save code by avoiding:
```
catch(e){
    const message = e instanceof Error ? e.message : 'Custom error message'
    sendToSentry(new Error(message))
```
in each catch that has a sendToSentry call but it looks like that's the best bet